cpp/libcudf_kafka/tests/kafka_consumer_tests.cpp

```diff
@@ -32,25 +32,37 @@ TEST_F(KafkaDatasourceTest, MissingGroupID)
 {
   // group.id is a required configuration.
   std::map<std::string, std::string> kafka_configs;
-  kafka_configs.insert({"bootstrap.servers", "localhost:9092"});
+  kafka_configs["bootstrap.servers"] = "localhost:9092";
 
-  EXPECT_THROW(kafka::kafka_consumer kc(kafka_configs, "csv-topic", 0, 0, 3, 5000, "\n"),
-               cudf::logic_error);
+  kafka::python_callable_type python_callable;
+  kafka::kafka_oauth_callback_wrapper_type callback_wrapper;
+
+  EXPECT_THROW(
+    kafka::kafka_consumer kc(
+      kafka_configs, python_callable, callback_wrapper, "csv-topic", 0, 0, 3, 5000, "\n"),
```

Comment:
Are the parameters here significant? Do we see failures with lower timeout etc? Just wondering.

Reply:
Not really significant. Any standard kafka installation uses those values by default so that is why I choose them. The librdkafka library is just lower level so you have to manually specify them where with in the Java Kafka library for example they just default to that.